### ISSUE TITLE
extracting dockets data in a date range using API call ?

### ISSUE THREAD

**Issue by purnima110895** [freelawproject/courtlistener#4049]
I am able to get the data for single date filtering -
https://www.courtlistener.com/api/rest/v3/dockets/?date_last_filing=2024-05-14

But the error arises when giving a date range -
https://www.courtlistener.com/api/rest/v3/dockets/?date_last_filing=2024-05-14TO2024-05-15

Please help.
@mlissner

**Comment by mlissner** [comment]
Have you reviewed the filtering documentation: https://www.courtlistener.com/help/api/rest/#filtering

**Comment by purnima110895** [comment]
Yes @mlissner , But in the documentation , I didn't find any date range filtering , is it possible to pass date range in the API call ?

**Comment by mlissner** [comment]
If you do an OPTIONS request to the endpoint, it'll tell you all the filters that are available. To do a range, simply use two filters on the same field, each with: `__gt` and `__lt`. 

For example, notionally:

```
GET https://pizzas.food/api/pizzas/?cook_date__gt=SOME_DATE&cook_date__lt=SOME_OTHER_DATE
```

Does that help?
